### PR TITLE
Improvements and typo fixes for BRD4:inhibitor benchmark

### PR DESCRIPTION
**Pull request by jchodera**
This PR refines the BRD4:inhibitor benchmark example notebook, fixing some embarrassing typos.

### CODE REVIEW

File: examples/using_smirnoff_with_amber_protein_forcefield/BRD4_inhibitor_benchmark.ipynb [hunk 1]
```diff
@@ -172,9 +168,9 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "### Export to OpenMM, Amber, and gromacs\n",
+    "### Export to OpenMM\n",
     "\n",
-    "Once we have the `Structure` of the complex, we can chose to create a `System` object that we can simulate with OpenMM:"
+    "Once we have the `Structure` for the complex, containing both positions and parameters, we can chose to create an OpenMM `System` object that we can simulate using [OpenMM](http://openmm.org):"
```
Comment (j-wags) [inline examples/using_smirnoff_with_amber_protein_forcefield/BRD4_inhibitor_benchmark.ipynb]: chose --> choose

File: examples/using_smirnoff_with_amber_protein_forcefield/BRD4_inhibitor_benchmark.ipynb [hunk 2]
```diff
@@ -69,25 +64,22 @@
...
      "output_type": "stream",
      "text": [
+      "Unable to load toolkit <openforcefield.utils.toolkits.OpenEyeToolkitWrapper object at 0x11082c0b8>.\n",
       "In AmberToolsToolkitwrapper.computer_partial_charges_am1bcc: Molecule 'BRD4, Ligand 1' has more than one conformer, but this function will only generate charges for the first one.\n"
      ]
     }
    ],
    "source": [
-    "# Create an OpenFF Topology of the ligand from the SDF file\n",
+    "# Create an OpenFF Molecule object from the ligand SDF fiel\n",
```
Comment (andrrizzi) [inline examples/using_smirnoff_with_amber_protein_forcefield/BRD4_inhibitor_benchmark.ipynb]: One last typo: fiel -> file

**Comment by jchodera** [comment]
Fixed, thanks!

File: examples/using_smirnoff_with_amber_protein_forcefield/BRD4_inhibitor_benchmark.ipynb (hunk 2 above)
Comment (jchodera) [reply examples/using_smirnoff_with_amber_protein_forcefield/BRD4_inhibitor_benchmark.ipynb]: Fixed, thanks!

File: examples/using_smirnoff_with_amber_protein_forcefield/BRD4_inhibitor_benchmark.ipynb (hunk 1 above)
Comment (jchodera) [reply examples/using_smirnoff_with_amber_protein_forcefield/BRD4_inhibitor_benchmark.ipynb]: Fixed, thanks!

**Pull request merged**